src/s2/util/math/exactfloat/exactfloat.cc

```diff
@@ -285,16 +201,15 @@ ExactFloat ExactFloat::RoundToPowerOf2(int bit_exp, RoundingMode mode) const {
     //    0/10*       ->    Don't increment (fraction = 1/2, kept part even)
     //    1/10*       ->    Increment (fraction = 1/2, kept part odd)
     //    ./1.*1.*    ->    Increment (fraction > 1/2)
-    if (BN_is_bit_set(bn_.get(), shift - 1) &&
-        ((BN_is_bit_set(bn_.get(), shift) ||
-          BN_ext_count_low_zero_bits(bn_.get()) < shift - 1))) {
+    if (bn_.is_bit_set(shift - 1) &&
+        ((bn_.is_bit_set(shift) || countr_zero(bn_) < shift - 1))) {
       increment = true;
     }
   }
   r.bn_exp_ = bn_exp_ + shift;
-  ABSL_CHECK(BN_rshift(r.bn_.get(), bn_.get(), shift));
+  r.bn_ = bn_ >> shift;
   if (increment) {
-    ABSL_CHECK(BN_add_word(r.bn_.get(), 1));
+    r.bn_ += Bignum(1);
```

Comment:
The old code used `BN_add_word`. Would `operator+=(Bigit)` help, or is this the only place it would be used and it's irrelevant?